lib/twitch-soundboard/static/css/soundboard.css

```diff
@@ -1,4 +1,92 @@
+.divider1 {
+  float: left;
+  width: 30%;
+  margin: 1em;
+  border: solid black 10px;
+}
+
+.title1 {
+  height: 40px;
+  font-size: 30px;
+  color: blue;
+}
+
 button#gift {
```

Comment:
Can you consolidate the button styling to a class or apply it to the specific button tag? It looks like they're all the same.